src/dcap_provider.cpp

```diff
@@ -54,12 +54,15 @@ static const std::map<std::string, std::string> default_values = {
 // New API version used to request PEM encoded CRLs
 constexpr char API_VERSION_LEGACY[] = "api-version=2018-10-01-preview";
 constexpr char API_VERSION[] = "api-version=2020-02-12-preview";
-
 static char DEFAULT_CERT_URL[] =
-    "https://global.acccache.azure.net/sgx/certificates";
+    "https://global.acccache.azure.net/sgx/certificates/";
 static std::string cert_base_url = DEFAULT_CERT_URL;
 
-static char DEFAULT_CLIENT_ID[] = "production_client";
+static char DEFAULT_THIM_AGENT_URL[] =
+    "http://169.254.169.254/metadata/THIM/sgx/getCerts?";
+static std::string thim_agent_base_url = DEFAULT_THIM_AGENT_URL;
+
+static char DEFAULT_CLIENT_ID[] = "Azure DCAP_client";

```

Comment:
Why are we changing the client ID? Originally it was `production_client`.

Reply:
I updated this for telemetry purposes. I think it will be easier to summarize the result.

Reply:
Gotcha. Be careful with spaces in the URL; otherwise should be fine